flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/sink/KafkaCommittableSerializerTest.java

```diff
@@ -29,7 +30,8 @@
  * Tests for serializing and deserialzing {@link KafkaCommittable} with {@link
  * KafkaCommittableSerializer}.
  */
-public class KafkaCommittableSerializerTest extends TestLogger {
+@ExtendWith(TestLoggerExtension.class)
+public class KafkaCommittableSerializerTest {
```

Comment:
```suggestion
class KafkaCommittableSerializerTest {
```
could be package private